scripts/setup/enable-pgroonga

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+# What user should we use for connecting to the database
+POSTGRES_USER="${POSTGRES_USER:-postgres}"
+
+(
+# Make sure the current working directory is readable by postgres
+cd /
+
+su "$POSTGRES_USER" -c 'psql zulip' <<EOF
+CREATE EXTENSION pgroonga;
+
+GRANT USAGE ON SCHEMA pgroonga TO zulip;
+ALTER ROLE zulip SET search_path TO zulip,public,pgroonga,pg_catalog;
+
+SET search_path = zulip,public,pgroonga,pg_catalog;
+
+ALTER TABLE zerver_message ADD COLUMN search_pgroonga text;
+
+UPDATE zerver_message SET search_pgroonga = subject || ' ' || rendered_content;
+
+CREATE INDEX zerver_message_search_pgroonga ON zerver_message
```

Comment:
We probably want this to use `CREATE INDEX CONCURRENTLY`; otherwise, nobody will be able to send messages while this is running.